### DISCUSSION TITLE
Auto call-back creation.

### DISCUSSION THREAD

**Discussion by Natanel-Shitrit** [SPCodeOrg/SPCode#62] [Ideas]
## **General Idea**
When using commands like: `RegConsoleCmd()`, a callback of `ConCmd` will be created automatically, and when typing the callback name in the command it will add it to the created callback too.

**Comment by Hexer10** [comment]
We should also think about how to handle cases when you have multiple definitions for a callback, look at `SDKHook` for example, maybe popping-up the autocomplete box could be a solution.
